content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -17,9 +17,9 @@ toc::[]
 :icons: font
 :imagesdir: /images/documentation/runtimes-monitoring/
 
-Kiali can display custom dashboards to monitor application metrics. They are available for Applications and Workloads.
+Kiali can display custom dashboards to monitor application metrics. They are available for applications and workloads.
```

Comment:
Keep the capitalization, it matches the navigation in the UI (custom dashboards are available on the Applications and Workloads screens).